src/main/java/io/debezium/connector/yugabytedb/consistent/Merger.java

```diff
@@ -88,4 +162,62 @@ public boolean isEmpty() {
     public boolean isSlotEmpty(String tabletId) {
         return mergeSlots.get(tabletId).isEmpty();
     }
+
+    /**
+     * Whether the messages in the merge slot are sorted properly by the virtue of their
```

Comment:
Looks like this is unused. Keep it or remove it?

Reply:
Let's keep it for sometime, if required we will remove it around the final cleanup.